e2etest/bgptests/bgp.go

```diff
@@ -126,6 +126,19 @@ var _ = ginkgo.Describe("BGP", func() {
 
 			svc, _ := testservice.CreateWithBackend(cs, f.Namespace.Name, "external-local-lb", tweak)
 			defer testservice.Delete(cs, svc)
+			desiredLbIPs := svc.Annotations["metallb.universe.tf/loadBalancerIPs"]
+			if desiredLbIPs != "" {
+				ingressPts := svc.Status.LoadBalancer.Ingress
+				framework.ExpectNotEqual(len(ingressPts), 0)
```

Comment:
no need for this check as you are comparing below.

Reply:
yes, done.